### PR TITLE
useful-programs: Mention ayatana-webmail

### PR DESCRIPTION
**Pull request by doronbehar**
It's the best modern mail notification application I've seen, and it's not well known enough IMO.

### CODE REVIEW

File: _contrib/useful-programs.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ See also: [external address queries](/guide/advancedusage.html#query).
   user of something without interrupting their work with a dialog box that they
   must close. Passive pop-ups can automatically disappear after a short period
   of time.
+* [ayatana-webmail](https://github.com/AyatanaIndicators/ayatana-webmail) - A
+  desktop Notification daemon that can watch imap mailboxes using `IDLE`. Can
+  also run sync commands when a new message arrives. See also
+  [this patch](https://github.com/AyatanaIndicators/ayatana-webmail/pull/33).
```
Comment (flatcap) [inline _contrib/useful-programs.md]: Mind if we drop this last sentence about the patch?
It's just been merged.

**Comment by doronbehar** [comment]
Yes it got merged, but there is no release yet that includes that patch.

**Comment by flatcap** [comment]
Yeah, but they've only made one release and that was nearly three years ago.

I'm happy to accept links to any useful programs, but linking to patches seems a bit much.
I suggest poking Ayatana into making a release, then everyone benefits.

**Comment by doronbehar** [comment]
I too thought they haven't made a release in 3 years, but they did made git tags. In anycase, thinking about it, that's not an important detail, as this feature is still useful - whether you have multiple mbsync accounts or not. I removed that last sentence.

**Review by flatcap — APPROVED** [review]
Thanks!

**Pull request merged**